Languages/Dutch/Keyed/Multiplayer.xml

```diff
@@ -25,18 +25,18 @@
   <MpInvalidUsernameAlreadyPlaying>Gebruiker is al aanwezig in de game</MpInvalidUsernameAlreadyPlaying>
   <MpInvalidUsernameChars>Gebruikersnaam bevat ongeldige tekens</MpInvalidUsernameChars>
   <MpInvalidUsernameLength>Gebruikersnaam heeft een ongeldige lengte</MpInvalidUsernameLength>
-  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
+  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de mod instellingen.</MpChangeUsernameInfo>
   
   <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
   <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
   
   <MpReplayOutdated>"Niet up-to-date"</MpReplayOutdated>
-  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
-  <MpReplayOutdatedDesc2>De herhalingen worden mogelijks niet correct afgespeeld.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar jij gebruikt versie {1}.</MpReplayOutdatedDesc1>

```

Comment:
"u" i.p.v. "jij"

Reply:
I think you had a stroke bud

Reply:
Does he just want to change MpReplayOutDatedDesc1? or Change it and MpReplayOutDatedDesc2? I'm assuming its a mistype.

Reply:
Not sure we need him to respond first.

Reply:
My requested changes still need to be applied, the way it is formulated right now is the "impolite way", and it needs to be written in the "polite way" (I don't know how it's called in English).